### DISCUSSION TITLE
Issue with SSP Installation - Interface Not Available

### DISCUSSION THREAD

**Discussion by marianezei** [ltb-project/self-service-password#963] [Q&A] [answered]
Hi! I'm having some difficulties with the installation of SSP. I tried configuring it with both Apache and NGINX, but in both cases, the interface doesn't become available at all. In the documentation [here](https://self-service-password.readthedocs.io/en/latest/config_nginx.html), there is an example of the php.ini file. Where exactly should I create it?

P.S.: The ports are properly opened on the VM.

**Comment by coudot** [comment, marked as answer]
You should first enable debug and check for error messages

https://self-service-password.readthedocs.io/en/stable/config_general.html#debug
